cmd/tsconnect/index.html

```diff
@@ -12,7 +12,27 @@ <h1 class="text-3xl font-bold grow">Tailscale Connect</h1>
         <div class="text-gray-600" id="state">Loading…</div>
       </header>
     </div>
-    <div id="peers" class="container mx-auto px-4"></div>
+    <form
+      id="ssh-form"
+      class="container mx-auto px-4 hidden flex justify-center"
+    >
+      <input type="text" class="input username" placeholder="Username" />
+      <div class="select-with-arrow mx-2">
+        <select class="select"></select>
+      </div>
+      <input
+        type="submit"
+        class="button bg-green-500 border-green-500 text-white hover:bg-green-600 hover:border-green-600"
+        value="SSH"
+      />
+    </form>
+    <div id="no-ssh" class="container mx-auto px-4 hidden text-center">
+      None of your machines have
+      <a href="https://tailscale.com/kb/1193/tailscale-ssh/" class="link"
+        >Tailscale SSH</a
+      >
+      enabled. Give it a try!
+    </div>
```

Comment:
Might be known behaviour but this is being triggered when a machine isn't auth'd and retains the QR/login link below. Even though it says Needs Auth top right, it might be a bit confusing. I don't think this was happening prior but might be wrong.

<img width="1606" alt="Screen Shot 2022-07-29 at 3 03 16 PM" src="https://user-images.githubusercontent.com/46385858/181849952-97f9f25b-2086-4c74-9332-e2b8311245da.png">

Reply:
Don't think that's a regression(we had a similar empty check for the list of SSH peers), but I agree that it's a bug. Added a check on the Go side to not send the netmap notification if we're not in the `Running` state.